### ISSUE TITLE
Add ARM64 binaries and deb package

### ISSUE THREAD

**Issue by olkitu** [firecow/gitlab-ci-local#704]
Possible run multi-arch image to build example amd64 and arm64. 

docker support this by using `--platform` attribute and this could be used in job level looking variable `DOCKER_DEFAULT_PLATFORM` to override default one.

**Comment by firecow** [comment]
This tools isn't published as a docker image.

But we could add arm64 binaries.

`npm install -g gitlab-ci-local` will be arm64 native experience.

**Comment by olkitu** [comment]
Yes but it use docker to run jobs and when use `--platform` could run builds with different arch than host system (example use qemu). Example I build always arm64 and amd64 versions of images with Gitlab CI and like to use gitlab-ci-local to test jobs without having custom jon to run locally only.

**Comment by firecow** [comment]
I don't quite understand....

**Comment by olkitu** [comment]
Sorry, seems there is no issues when I made test project to build test image with arm64 and amd64: https://gitlab.com/olkitu/gitlab-ci-local-test . My test project build multi-arch image properly when set --platform inside of the container.  Thank you for great project. 

Maybe close this and open another issue just arm64 binaries only if like to have in repository?

**Comment by firecow** [comment]
Nah, leave it this open, I've renamed the issue..

**Comment by firecow** [comment]
https://github.com/firecow/gitlab-ci-local/issues/1242#issuecomment-2143510105

**Comment by firecow** [comment]
https://github.com/firecow/gitlab-ci-local/actions/runs/9331763450/job/25686810204?pr=1243

I don't think we can get Github Actions to-do it either :smile:

**Comment by firecow** [comment]
https://github.com/firecow/gitlab-ci-local/actions/runs/9331830871/job/25686953975

Here is me using the https://github.com/lando/pkg-action action

**Comment by firecow** [comment]
This has been fixed
